package.json

```diff
@@ -199,7 +199,6 @@
     "eslint-plugin-adeira": "^0.15.0",
     "eslint-plugin-angular": "^4.1.0",
     "eslint-plugin-awscdk": "^0.0.65",
-    "eslint-plugin-babel": "^5.3.1",
```

Comment:
is this intended?

Reply:
Yes:
https://github.com/babel/eslint-plugin-babel#note-eslint-plugin-babel-is-now-babeleslint-plugin-and-has-moved-into-the-babel-monorepo
It was also in an old open PR (which I've closed):
https://github.com/codacy/codacy-eslint/pull/2413